src/batteryinfoframe.cpp

```diff
@@ -62,8 +62,8 @@ BatteryInfoFrame::~BatteryInfoFrame()
 void BatteryInfoFrame::onBatteryChanged()
 {
     mUi->stateValue->setText(BatteryHelper::stateToString(mBattery->chargeState()));
-    mUi->energyFullValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg(mBattery->energyFull(), 0, 'f', 2).arg(mBattery->capacity()));
-    mUi->energyValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg(mBattery->energy(), 0, 'f', 2).arg(mBattery->chargePercent()));
+    mUi->energyFullValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg( mBattery->energyFull(), mBattery->capacity(), 0, 'f', 2 ));
+    mUi->energyValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg(mBattery->energy(), mBattery->chargePercent(), 0, 'f', 2));
```

Comment:
Oh, my bad! These are incorrect. I wonder how many instances I missed in other places. Should check all of them :(

Reply:
Not sure I understood the problem here...

Reply:
See https://doc.qt.io/qt-5/qstring.html#arg-14 (and uts following methods). Here we have numbers, not strings.

Reply:
oh... now I see. Sorry. Reverting those lines anche checking the other ones.